go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/hashicorp/golang-lru v0.5.3 // indirect
 	github.com/json-iterator/go v1.1.8 // indirect
 	github.com/kr/pty v1.1.8 // indirect
-	github.com/mattn/go-isatty v0.0.10
+	github.com/mattn/go-isatty v0.0.11
```

Comment:
oops, that's unrelated.

Reply:
yeah - I keep getting the same staged changes locally too but I was not checking it in. Since your getting it too might as well check it in to avoid the hassle.

Reply:
Or we could merge https://github.com/submariner-io/submariner-operator/pull/136 ...